tests/parsers/test_rinex_obs.py

```diff
@@ -241,3 +238,19 @@ def test_rinex_obs_3_fails(rinex_mixed_values):
         if count == 1:
             sv_not_pseudo = rinex_frame.where('gnss_sv_id', 'G29', 'eq')
             assert len(sv_not_pseudo) == 0
+
+def test_z_tracking(root_path):
+    """Test z-tracking support.
+
+    Parameters
+    ----------
+    root_path : string
+        Folder location containing all Rinex 3 .o files for unit tests.
+
+    """
+    rinex_path = os.path.join(root_path,
+                              'z_tracking.rnx')
+    rinex_navdata = RinexObs(rinex_path)
+
+    assert len(rinex_navdata) == 210
+    assert rinex_navdata.shape == (10,210)
```

Comment:
These two assert statements may leave open the case where we parse the requested file, but the z_tracking data are not correctly included in the `rinex_navdata` instance. Perhaps we can add a test case where we check that the measurements in `rinex_navdata` include the expected signal types. We could also "spot check" the `rinex_navdata` instance to verify that select measurements are correctly included.